### PR TITLE
Use GDC builtin math functions in mir.math.common

### PR DESCRIPTION
**Pull request by n8sh**
See:
https://github.com/D-Programming-GDC/GDC/blob/master/libphobos/libdruntime/gcc/builtins.d
https://gcc.gnu.org/git/?p=gcc.git;a=blob;f=gcc/builtins.def;hb=HEAD

### CODE REVIEW

**Comment by n8sh** [comment]
Code coverage is failing because none of the test machines use GDC.

File: source/mir/math/common.d [hunk 1]
```diff
@@ -217,11 +217,83 @@ version(LDC)
...
+}
 else
 {
     static import std.math;
+    static import core.math;
     ///
-    T sqrt(T)(in T x) if (isFloatingPoint!T) { return std.math.sqrt(x); }
+    T sqrt(T)(in T x) if (isFloatingPoint!T) { return core.math.sqrt(x); }
     ///
     T sin(T)(in T x) if (isFloatingPoint!T) { return std.math.sin(x); }
```
Comment (9il) [inline source/mir/math/common.d]: `std` is used here. Could you please add a small unittest, one function should be ok, and GDC to travis with this PR?

Comment (n8sh) [reply source/mir/math/common.d]: Sure

Comment (n8sh) [reply source/mir/math/common.d]: Uninttest added. I was wrong about travis: GDC is being tested. But "latest gdc" on travis has frontend version 2.068.2 so most of mir can't compile.

**Pull request merged**